legend-engine-language-pure-compiler/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/MilestoningDatePropagationHelper.java

```diff
@@ -68,24 +69,38 @@ public static void setMilestoningDates(ListIterable<? extends ValueSpecification
     {
         if (isBusinessTemporal(milestoningStereotype))
         {
-            processingContext.milestoningDatePropagationContext.setBusinessDate(temporalParameterValues.get(1));
+            ValueSpecification businessdate = null;
+            if (temporalParameterValues.size() > 1)
```

Comment:
Checking if parameters are **_greater than_** seems odd to me.  In what case defining more than expected is correct?  

Even further, if the modeled class does not have any temporal stereotype, should we validate for zero parameters?

I recall seeing users defining temporal argument on non-temporal classes without any failure.